### PR TITLE
improve reference text to secure storage connector

### PR DESCRIPTION
**Pull request by mkaesz-wandb**
## Description

The docs include a description on team level buckets at two places. This PR changes the first place (https://docs.wandb.ai/guides/app/features/teams#secure-storage-connector) to point to the second place (https://docs.wandb.ai/guides/hosting/data-security/secure-storage-connector) that is up to date.

## Checklist

Check if your PR fulfills the following requirements. Put an `X` in the boxes that apply.

- [X] Files I edited were previewed on my local development server with `yarn start`. My changes did not break the local preview.
- [X] Build (`yarn docusaurus build`) was run locally and successfully without errors or warnings.
- [X] I merged the latest changes from `main` into my feature branch before submitting this PR.

### CODE REVIEW

**Review by abhinavg6 — CHANGES_REQUESTED** [review]
Just requested a minor change.

**Review by mkaesz-wandb — COMMENTED** [review]
Thank you!

File: docs/guides/app/features/teams.md [hunk 1]
```diff
@@ -154,24 +154,4 @@ You can see the privacy settings of all team projects on the team settings page:
...
-
-A cloud storage bucket configures only once for a team at the time of team creation. Select **External Storage** when you create a team to configure a cloud storage bucket. Select your provider and fill out your bucket name and storage encryption key ID, if applicable, and select **Create Team**.
-
-An error or warning appears at the bottom of the page if there are issues accessing the bucket or the bucket has invalid settings.
-
-![](/images/hosting/saas_setup_secure_storage.png)
-
-Only organization administrators have the permissions to configure the secure storage connector. The same cloud storage bucket is accessible by multiple teams by selecting an existing cloud storage bucket from the dropdown.
+The team-level secure storage connector allows teams to use their own cloud storage bucket with W&B. This provides greater data access control and data isolation for teams with highly sensitive data or strict compliance requirements. See the [Secure Storage Connector](../../hosting/data-security/secure-storage-connector.md) page for more information.
```
Comment (abhinavg6) [inline docs/guides/app/features/teams.md]:
```suggestion
The team-level secure storage connector allows teams to use their own cloud storage bucket with W&B. This provides greater data access control and data isolation for teams with highly sensitive data or strict compliance requirements. Refer to [Secure Storage Connector](../../hosting/data-security/secure-storage-connector.md) for more information.
```

**Review by abhinavg6 — CHANGES_REQUESTED** [review]
Requesting one nit change

**Review by abhinavg6 — APPROVED** [review]
LGTM, thank you

**Review by ngrayluna — APPROVED** [review]

**Pull request merged**